packages/trader/src/Modules/Trading/Components/Form/TradeParams/__tests__/trade-type-tabs.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import TradeTypeTabs from '../trade-type-tabs';
+import { useStore } from '@deriv/stores';
+
+const mocked_root_store: Partial<ReturnType<typeof useStore>> = {
```

Comment:
@maryia-deriv Please use the `StoreProvider` and the `mockStore` for testing components using the `useStore` hook 🙏🏻

Reply:
Sure, I'll try to implement it in tests